File: Edad.php
<?php

echo"Ingrese su fecha de nacimiento\n";
$dia = readline("Día: ");
$mes = readline("Mes: ");
$anno = readline("Año: ");
$HoyDia = date("d");
$HoyMes = date("m");
$HoyAnno = date("Y");

if($mes < $HoyMes){
    $resultado = abs($anno - $HoyAnno);
    echo "Usted tiene $resultado años";
}elseif($mes > $HoyMes){
    $resultado = abs($anno - $HoyAnno) - 1;
    echo "Usted tiene $resultado años";
}elseif($mes == $HoyMes && $dia > $HoyDia){
    $resultado = abs($anno - $HoyAnno) - 1;
    echo "Usted tiene $resultado años";
}elseif($mes == $HoyMes && $dia < $HoyDia || $anno != $HoyAnno){
    $resultado = abs($anno - $HoyAnno);
    echo "Usted tiene $resultado años";
}else {
    echo "Usted tiene cero años";
}

File: Edad.py
from datetime import datetime

print("Ingrese su fecha de nacimiento")
dia = int(input("Día: "))
mes = int(input("Mes: "))
anno = int(input("Año: "))

fecha = datetime.now()

if mes < fecha.month:
    resultado = abs(anno - fecha.year)
elif mes > fecha.month:
    resultado = abs(anno - fecha.year) - 1
elif mes == fecha.month and dia > fecha.day:
    resultado = abs(anno - fecha.year) - 1
elif mes == fecha.month and dia < fecha.day or anno != fecha.year:
    resultado = abs(anno - fecha.year)
else:
    resultado = 0
    
print("Usted tiene", resultado, "años")



